latest/swagger-doc/v2/properties.json

```diff
@@ -1,4 +1,4 @@
 {
   "gaffer.properties.app.title": "Gaffer REST API",
-  "gaffer.properties.app.doc.url": "https://gchq.github.io/gaffer-doc/"
-}
\ No newline at end of file
+  "gaffer.properties.app.doc.url": "https://gchq.github.io/gaffer-doc/v1docs/"
```

Comment:
What does this do?

Reply:
See other comment. The `latest` and `v1docs` folders are identical. The latest folder is what is served by default, once v2 docs are deployed, only `v1docs` will contain the old docs.